pkg/command/promote.go

```diff
@@ -3,6 +3,7 @@ package command
 import (
 	"context"
 	"fmt"
+	"log"
```

Comment:
Is there a difference between creating a logger in main and passing it using composition vs. just importing it in each package?

Reply:
Not really. There is a "default" logger instance https://cs.opensource.google/go/go/+/refs/tags/go1.17.3:src/log/log.go;l=76 which you can use.